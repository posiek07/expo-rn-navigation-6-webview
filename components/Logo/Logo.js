import React from 'react'
import styled from 'styled-components'

const StyledCardImage = styled.Image`
  width: 100%;
  height: 30%;
`;

const Logo = ({logo}) => {
    return (
        <StyledCardImage source={{uri: logo}} />
    )
}

export default Logo

