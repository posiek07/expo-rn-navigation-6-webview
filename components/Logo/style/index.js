import styled from 'styled-components';

export const StyledCardImage = styled.Image`
  width: 100%;
  height: 30%;
`;
