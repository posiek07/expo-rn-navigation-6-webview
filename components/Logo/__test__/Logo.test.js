import Logo from '../Logo';
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components/native';

it('Logo should render and match snapshot', async () => {
  const url = 'https://picsum.photos/200'; // random image
  const logoImage = renderer.create(<Logo logo={url} />).toJSON();
  expect(logoImage).toMatchSnapshot();
});
