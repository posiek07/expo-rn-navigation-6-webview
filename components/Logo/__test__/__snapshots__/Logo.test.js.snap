// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render and match snapshot 1`] = `
<Image
  source={
    Object {
      "uri": "https://picsum.photos/200",
    }
  }
  style={
    Array [
      Object {
        "height": "30%",
        "width": "100%",
      },
    ]
  }
/>
`;
